presto-docs/src/main/sphinx/security/server.rst

```diff
@@ -124,6 +125,9 @@ Property                                                  Description
                                                           authentication is denied.  Default is ``(.*)``.
 ``http-server.authentication.krb5.user-mapping.file``     File containing rules for mapping user.  See
                                                           :doc:`/security/user-mapping` for more information.
+``node.internal-address-source``                          Kerberos is typically sensitive to DNS names. Setting this
```

Comment:
Please keep text inside the table boundaries

Reply:
Fixed, has to effect on output btw.